native/effektio/src/api/messages.rs

```diff
@@ -33,6 +39,91 @@ impl RoomMessage {
     pub fn origin_server_ts(&self) -> u64 {
         self.inner.origin_server_ts.as_secs().into()
     }
+
+    pub fn msgtype(&self) -> String {
+        self.inner.content.msgtype().to_string()
+    }
+
+    pub async fn image_description(&self) -> Result<ImageDescription> {
+        match &self.inner.content.msgtype {
+            MessageType::Image(content) => {
+                let client = Client::current_client().unwrap();
```

Comment:
rather than assuming a global, we should put `client` onto the room-message (and the image description), so we can do:

   ```suggestion
                let client = self.client.clone();
```